packages/gatsby/src/cache-dir/production-app.js

```diff
@@ -18,24 +18,40 @@ window.matchPath = matchPath
 import requires from "./async-requires"
 
 // Find page
-const findPage = pathname =>
-  pages.find(page => {
+const findPage = pathname => {
+  let foundPage
+  // Array.prototype.find is supported in IE so we use this
+  // weird work around.
+  pages.some(page => {
```

Comment:
Extra weight. Core should be as slim as possible so rely on browser primitives whenever possible.